records_mover/airflow/hooks/sqlalchemy_db_hook.py

```diff
@@ -6,7 +6,7 @@
     from airflow.hooks import BaseHook
 except ImportError:
     # Required for Airflow 2.0
-    from airflow.hooks.base_hook import BaseHook  # type: ignore
+    from airflow.hooks.base import BaseHook  # type: ignore
```

Comment:
I think this whole block should be simplified now that we use airflow >= 2.0

Reply:
@Brunope ack, missed this comment, good point, will fix this before merge.